packages/ui/react/src/components/Primitives/ArrowHead/ArrowHead.tsx

```diff
@@ -0,0 +1,37 @@
+import { Container, ArrowShape } from './ArrowHead.styled'
+
+type ArrowHeadProps = {
+  /** Fill color for the arrow shape */
+  fill: string
+  /** Stroke color for the arrow shape border */
+  stroke: string
+  /** Optional CSS styles to apply to the arrow shape */
+  style?: React.CSSProperties
+  /** Optional click handler for the arrow */
```

Comment:
Can you update the prop comments?

```suggestion
  /** CSS styles to apply to the arrow shape */
  style?: React.CSSProperties
  /**
   * Event handler called when the arrow is clicked.
   */
```